### PR TITLE
Remove the title tag from the generated HTML

### LINKED ISSUE #5347: HTML Copy sometimes inserts “Windows Terminal”; perhaps this is the TITLE element
# Description of the new feature/enhancement

Now, when I copy texture in a windows terminal, and then paste it in some place like discourse comments, the prefix "Windows Terminal" will added. e.g. if I copy "hello", what will paste in discoure is:
```
Windows Terminal

hello
```
which is quite troublesome, because I need delete this prefix by hand.
What I want is to remove prefix and only texture be pasted.
I also noticed that when paste in some place (like here), the prefix is gone. So this behavior may related to the place where to pasted. 
 # Proposed technical implementation details (optional)

### PR DESCRIPTION
**Pull request by zadjii-msft**
## Summary of the Pull Request

Web apps apparently will paste the \<title\> as plaintext before the actual HTML content from the clipboard. Since this seems to be widespread behavior across web apps, this isn't just a bug in _some app_, this is a bug on us. We shouldn't emit the title. 

This PR removes the title tag from the generated HTML.

## PR Checklist
* [x] Closes #5347
* [x] I work here
* [ ] Tests added/passed
* [n/a] Requires documentation to be updated

## Validation Steps Performed

Above is composing an email with copied text from 0.11.
Below is using the same webmail client to compose an email with this patch
image

### CODE REVIEW

File: src/interactivity/win32/Clipboard.cpp [hunk 1]
```diff
@@ -277,7 +277,10 @@ void Clipboard::CopyTextToSystemClipboard(const TextBuffer::TextAndColor& rows,
             int const iFontHeightPoints = fontData.GetUnscaledSize().Y * 72 / ServiceLocator::LocateGlobals().dpi;
             const COLORREF bgColor = ServiceLocator::LocateGlobals().getConsoleInformation().GetDefaultBackground();
 
-            std::string HTMLToPlaceOnClip = TextBuffer::GenHTML(rows, iFontHeightPoints, fontData.GetFaceName(), bgColor, "Windows Console Host");
+            // GH#5347 - Don't provide a title for the generated HTML, as many
+            // web applications will paste the title first, followed by the HTML
+            // content, which is unexpected.
+            std::string HTMLToPlaceOnClip = TextBuffer::GenHTML(rows, iFontHeightPoints, fontData.GetFaceName(), bgColor, "");
```
Comment (DHowett-MSFT) [inline src/interactivity/win32/Clipboard.cpp:283]: should we go one step further and remove the parameter and the generation of the empty `<TITLE></TITLE>`?

Comment (carlos-zamora) [reply src/interactivity/win32/Clipboard.cpp:283]: Yeah, if this is incorrect behavior, do that. That'll fix it for ConHost too then.

Comment (zadjii-msft) [reply src/interactivity/win32/Clipboard.cpp]: It's done

**Comment by DHowett-MSFT** [comment]
Needs-build-fix

**Comment by vadimkantorov** [comment]
I don't know if it's an issue, but maybe head should specify a charset (be it utf-8 or sth else?)

**Comment by DHowett-MSFT** [comment]
admin merge: CI says code health passed.

**Pull request merged**